rook/migration/v1.12/README.md

```diff
@@ -0,0 +1,223 @@
+# Upgrade rook-ceph to v1.12.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Prepare upgrade - *non-disruptive*
+
+*Nothing to prepare*
+
+## Apply upgrade - *non-disruptive*
+
+## Rook Operator Upgrade
+
+The examples given in this guide upgrade a live Rook cluster running `v1.11.11` to
+the version `v1.12.11`. This upgrade should work from any official patch release of Rook v1.11 to any
+official patch release of v1.12.
+
+Let's get started!
+
+### Environment
+
+These instructions will work for as long the environment is parameterized correctly.
+Set the following environment variables, which will be used throughout this document.
+
+```console
+# Parameterize the environment
+export ROOK_OPERATOR_NAMESPACE=rook-ceph
+export ROOK_CLUSTER_NAMESPACE=rook-ceph
+```
+
+### **1. Update common resources and CRDs**
+
+!!! hint
```

Comment:
These `!!!` does not seem to render nicely on github. Can you check if those are supported but github markdown?

Reply:
Yep! The thing is that it shows differently on [http://elastisys.io/](http://elastisys.io/) than on GitHub so we have to do it like this as of yet. Me and @cristiklein looked at this before but we haven't found anything that works on both. It all boils down to "mkdocs-material" which only can handle the old "!!!" annotation.

Reply:
Hmmm, thats awesome!! Do we convert the rest of our documentation to follow this also? :thinking: @cristiklein @OlleLarsson

Reply:
The IO documentation will be updated eventually it looks like by the statement from ck, there is this [pr](https://github.com/elastisys/compliantkubernetes/pull/861) for recommending github style alerts. For the other repos we already follow github style.